database-schema/V5__revoke_reader_user_permissions.sql

```diff
@@ -0,0 +1,9 @@
+REVOKE ALL PRIVILEGES ON DATABASE events FROM reader;
+REVOKE CONNECT ON DATABASE events FROM reader;
+REVOKE ALL ON billing.fraud_events FROM reader;
+REVOKE ALL ON billing.billing_events FROM reader;
+REVOKE ALL ON audit.audit_events FROM reader;
+REVOKE ALL ON SCHEMA billing FROM reader;
+REVOKE ALL ON SCHEMA audit FROM reader;
+-- Production and staging only
+GRANT ALL ON billing.billing_idps FROM reader;
```

Comment:
Should this be `REVOKE` instead of `GRANT`?

Reply:
It will error. I don't think we care though. I'm just capturing these things here as a record of what we've done. When we do the database migrations piece we should look at bringing all the databases in line. That's my thoughts on the matter, at least!

Reply:
I've fixed the `GRANT` thing as well ... copy and paste error 😳